collectors/networkviewer.plugin/network_viewer_plugin.c

```diff
@@ -138,6 +138,18 @@ static void netdata_create_charts() {
     netdata_create_chart(NETWORK_VIEWER_FAMILY, NETWORK_VIEWER_CHART6, "Network viewer TCP active connections per port.", "active connections", "TCP");
 
     netdata_create_chart(NETWORK_VIEWER_FAMILY, NETWORK_VIEWER_CHART8, "Network viewer UDP active connections per port.", "active connections","UDP");
+
+    netdata_create_chart(NETWORK_VIEWER_FAMILY, NETWORK_VIEWER_CHART9, "Network Viewer TCP bytes received from request to specific port.", "kilobits/s", "TCP");
```

Comment:
i think using some index instead of name is not a good idea. Long name > index imo.

> NETWORK_VIEWER_CHART9

> 9